src/components/WishListModal.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <div>
+       
+      <button
+      class="mt-4 bg-indigo-500 text-white py-2 px-6 rounded-md hover:bg-indigo-600"
+        @click="toggleModal = !toggleModal">Create Wishlist
+      </button>
```

Comment:
the create wishlist cannot be here, it should be in the `SingleExpenseCategory.vue` file